plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/TestHudiSmokeTest.java

```diff
@@ -469,17 +510,156 @@ public void testPartitionFilterRequiredOnJoinBothTablePartitioned()
     @Test
     public void testPartitionFilterRequiredWithLike()
     {
-        Session session = withPartitionFilterRequired(getSession());
+        Session session = SessionPropertyConfigurator.from(getSession())
+                .withPartitionFilterRequired(true)
+                .build();
         assertQueryFails(
                 session,
                 "SELECT name FROM " + HUDI_COW_PT_TBL + " WHERE name LIKE '%1'",
                 "Filter required on tests." + HUDI_COW_PT_TBL.getTableName() + " for at least one of the partition columns: dt, hh");
     }
 
+    @Test
+    public void testDynamicFilterEnabledPredicatePushdown()
+    {
+        Session session = SessionPropertyConfigurator.from(getSession())
+                .withMdtEnabled(false)
```

Comment:
The dynamic filtering works with and without MDT enabled.  Should we remove this config to turn of MDT and use default?

Reply:
It looks like we need to upgrade to Hudi 1.0.2 to safely turn on MDT for both table version 6 and 8.  So leaving this as a follow-up.